src/Model/Item.php

```diff
@@ -766,11 +725,8 @@ public static function insert($arr, $force_parent = false, $notify = false, $don
 
 		// How much entries have we created?
 		// We wouldn't need this query when we could use an unique index - but MySQL has length problems with them.
-		$r = q("SELECT COUNT(*) AS `entries` FROM `item` WHERE `uri` = '%s' AND `uid` = %d AND `network` = '%s'",
-			dbesc($arr['uri']),
-			intval($arr['uid']),
-			dbesc($arr['network'])
-		);
+		$r = dba::inArray(dba::p("SELECT COUNT(*) AS `entries` FROM `item` WHERE `uri` = ? AND `uid` = ? AND `network` = ?",
```

Comment:
Don't we have a `dba::count()` method?

Reply:
I guess I started that item class when this function hadn't existed. It is changed now.